### PR TITLE
Add eui-event-template

### PR DESCRIPTION
**Pull request by brittanyjoiner15**
I'd like to include my project here! It has several simple issues and is very beginner friendly!

### CODE REVIEW

**Comment by brittanyjoiner15** [comment]
Hmm, im a little confused what is wrong with the link? Is it expecting a blank URL? I tried out https://github.com/brittanyjoiner15/eui-event-template/labels/up%20for%20grabs and it does seem to go to my project and the issues with the `up for grabs` label. If you can clarify what I need to change, I'm happy to do so!

**Comment by itsfaizankhan** [comment]
Hello, the link you're using actually leads to the issues section of your project repository. Try using the code section's link i.e. https://github.com/brittanyjoiner15/eui-event-template
This should work.

**Comment by brittanyjoiner15** [comment]
@ATOMICMAN007 ah okay i misunderstood what i was supposed to put in that link. thanks!

File: _data/projects/eui-event-template.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+name: React and Eui Event Template
+desc: A React app to share information about an event or conference, such as speaker details, agenda, and event actions
+site: https://eui-event.vercel.app/
+tags:
+  - javascript
+  - reactjs
+  - beginner
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/brittanyjoiner15/eui-event-template
```
Comment (shiftkey) [inline _data/projects/eui-event-template.yml]: 🎨 this should be a deep link to the issues tagged with the `up for grabs` label:

```suggestion
  link: https://github.com/brittanyjoiner15/eui-event-template/labels/up%20for%20grabs
```

**Review by shiftkey — APPROVED** [review]
Thanks for the contribution. Don't worry about that message from the automation, that's on me to figure out why it's not happy...

**Pull request merged**